compilation/abiutils/solidity_errors.go

```diff
@@ -105,3 +105,35 @@ func GetSolidityCustomRevertError(contractAbi *abi.ABI, returnError error, retur
 	}
 	return nil, nil
 }
+
+// GetPanicReason will take in a panic code as an uint64 and will return the string reason behind that panic code. For
+// example, if panic code is PanicCodeAssertFailed, then "assertion failure" is returned. This is used by the execution
```

Comment:
Small nitpick, but with this comment, I wouldn't mention what it's used for here, since this is supposed to be agnostic from the fuzzing package, and it can be used for anything.

Reply:
Resolved in [cd27dcb](https://github.com/crytic/medusa/pull/161/commits/cd27dcb384e4f0ae2dc52f67534a6b329b1c1ec1)